examples/basic_geometries.py

```diff
@@ -0,0 +1,26 @@
+from src.gepetto_viewer_rerun.client import Client
```

Comment:
```suggestion
from gepetto_viewer_rerun import Client
```

Ideally, this is the API we would want I think.
But maybe this need a little more work to work like that.